Blueprints/deploy-microsoft-sql-server-using-blueprint.md

```diff
@@ -1,77 +1,88 @@
-</p>
-<p>3. Select Deploy blueprint.</p>
-<p><img src="https://t3n.zendesk.com/attachments/token/KOVvpyFsjYQRozwUYGqSWD5qj/?name=04.png" alt="04.png" />
-</p>
-<p>4. Input the appropriate parameters based on the SQL server requirements for your application and select the Virtual Machine you wish to execute the install against.</p>
-<p><img src="https://t3n.zendesk.com/attachments/token/hIyEiUPJRx0uJoYboE16mCc4O/?name=05.png" alt="05.png" />
-</p>
-<p>5. Confirm the virtual machine(s), features and select Deploy Blueprint</p>
-<p><img src="https://t3n.zendesk.com/attachments/token/skfISBior0Hg7nwnsYrqMEjOj/?name=06.png" alt="06.png" />
-</p>
+### Overview
+
+CenturyLink Cloud customers can procure and deploy Microsoft SQL Server licensing within the Control Portal. Microsoft SQL Server is licensed via the Microsoft SPLA program. By using the CenturyLink Cloud public blueprint customers have multiple ways to consume and install this business critical database software.
+
+### Prerequisites
+
+* A CenturyLink Cloud Account
+* Supported Guest Operating Systems are deployed and in a running state:
+    * Windows 2008 R2 Standard 64-bit
+    * Windows 2008 R2 Enterprise 64-bit
+    * Windows 2008 R2 Datacenter 64-bit
+    * Windows 2012 Datacenter 64-bit
+    * Windows 2012 R2 Data Center 64-bit
+* ~15 GB Free Storage on C:\
+* The Supported SQL Server Editions via blueprint are as follows:
+    * SQL Server 2008 R2 Web Edition
+    * SQL Server 2008 R2 Standard Edition
+    * SQL Server 2008 R2 Enterprise Edition
+    * SQL Server 2012 Web Edition
+    * SQL Server 2012 Standard Edition
+    * SQL Server 2012 Enterprise Edition
+    * SQL Server 2014 Web Edition
+    * SQL Server 2014 Standard Edition
+    * SQL Server 2014 Enterprise Edition
+
+### Exceptions
+
+This KB does not apply to [Managed Microsoft SQL Customers.](http://www.centurylinkcloud.com/managed-services/ms-sql)
+
+### General Notes
+
+The following are quick tips/notes based on past experiences with customers leveraging this blueprint
+
+* It is not possible at the current time to install SQL to a drive other than C:\ via blueprint. Customers can modify the SQL database, tempdb, log locations post install to other volumes using SQL tools
+* The fee's for Microsoft SQL server will be applied automatically to the customers invoice when using the public blueprint. These fee's are documented in a customers agreement. If you are unsure what these fee's are please contact your account manager.
+* Licensing fee's are adjusted based on number of vCPU allocated to a virtual machine. By using tools like Autoscale, customers billing will be modified as vCPU configurations change.
+
+### Installing Microsoft SQL Server using Group Tasks
+
+1. **Navigate to the Servers Menu in Control**
+
+<img src="../images/deploy-microsoft-sql-server-using-blueprint-01.png">
```

Comment:
If you preview the page in Github, you'll notice that every step is numbered "1.". This is because the images need to indented two spaces.